groups/sig-release/groups.yaml

```
@@ -106,6 +106,20 @@ groups:
       - linusa@google.com
       - stephen.k8s@agst.us
 
+  - email-id: k8s-infra-staging-ci-images@kubernetes.io
+    name: k8s-infra-staging-ci-images
+    description: |-
+      ACL for staging CI images
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - k8s-infra-release-editors@kubernetes
+      - ameukam@gmail.com
+      - bentheelder@google.com
+      - davanum@gmail.com
+      - linusa@google.com
+      - spiffxp@google.com
```

Ideally, this is instead:
- IAM for Test Infra folks
- IAM for WG K8s Infra Leads

---

I would recommend `k8s-infra-prow-oncall@kubernetes.io` instead
```suggestion
      - k8s-infra-prow-oncall@kubernetes.io
      - ameukam@gmail.com
```

I don't think @listx needs to be included here.  @ameukam I would support you moving to k8s-infra-prow-oncall if you're interested